packages/test-data/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cumulus/test-data",
-  "version": "1.11.0",
+  "version": "1.11.2",
```

Comment:
If we are arbitrarily updating the version, why wouldn't we update it to the latest version, `1.11.3`?

Reply:
I think because `1.11.2` was the latest version when I made that change.

Reply:
Cool. Can we update to `1.11.3` now then?